src/styles/block/card.scss

```diff
@@ -0,0 +1,70 @@
+.card {
+  width: 166px;
+  height: 358px;
```

Comment:
The same as above, fix it everywhere
```suggestion
  width: 100%;
```

Reply:
With such a condition, the element stretches across the entire width of the screen.